Legacy/bonej/src/main/java/org/bonej/plugins/UsageReporter.java

```diff
@@ -194,15 +190,16 @@ public static UsageReporter reportEvent(final String category,
 	}
 
 	/**
-	 * Prepare the instance for sending a report on a specific class; its name
-	 * (.getClass().getName()) is added to the 'action' field of the report,
-	 * category is "Plugin Usage" and label is the BoneJ version string
+	 * Prepare the instance for sending a report on a specific class; {@link Class#getName}
+	 * is added to the 'action' field of the report,
+	 * category is "Plugin Usage" and label is the version string generated by
+	 * {@link org.scijava.util.VersionUtils#getVersion}.
 	 *
 	 * @param o Class to report on
 	 * @return The instance of UsageReporter ready to send() a report
 	 */
 	public static UsageReporter reportEvent(final Object o) {
-		return reportEvent("Plugin%20Usage", o.getClass().getName(), BONEJ_VERSION,
+		return reportEvent("Plugin%20Usage", o.getClass().getName(), VersionUtils.getVersion(o.getClass()),
```

Comment:
Is the `Integer value` parameter in `reportEvent(String, String, Integer)` used anymore? The method is only called from `reportEvent(Object)`, where the argument is always `null`. Also, `reportEvent(String, String, Integer)` can be made `private`.

Reply:
> Is the `Integer value` parameter in `reportEvent(String, String, Integer)` used anymore?

I see the point. Keeping it for the moment so that we don't lose this bit of the Analytics API, even if it's not used at the moment.

Reply:
> What should happen if `VersionUtils.getVersion(o.getClass())` returns `null`?

In what case can that happen? An NPE would get thrown if `o == null`, which is fine.

Reply:
VersionUtils.getVersion(o.getClass()) returns a `null` when it fails to find a version number for the given class.